core/src/test/scala/org/apache/spark/SparkFunSuite.scala

```diff
@@ -106,4 +107,14 @@ abstract class SparkFunSuite
     }
   }
 
+  lazy val outstandingTimezones = Seq(
```

Comment:
Is this lazy so that it's not evaluated by each test suite? I get it although `TimeZone` caches these, it seems. It won't matter really either way. I'm neutral, it doesn't matter